app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -404,14 +404,9 @@ def save_custom_attribute_attribute_inventory(entity, attribute_name, hashes, ta
     return if hashes.nil?
 
     entity.send(attribute_name).reset
-    deletes = if target.kind_of?(ExtManagementSystem)
-                :use_association
-              else
-                []
-              end
 
     save_inventory_multi(entity.send(attribute_name),
-                         hashes, deletes, [:section, :name])
+                         hashes, :use_association, [:section, :name])
```

Comment:
Hi @cben, @simon3z 
Agreed. In fact, there are multiple places where we are re-using the containers tagging code, we will make a separate PR that will move it to a generic place.